src/registrar/management/commands/populate_organization_type.py

```diff
@@ -0,0 +1,208 @@
+            info_to_inspect=f"""
+            ==Proposed Changes==
+            Number of DomainInformation objects to change: {len(domain_infos)}
+
+            Organization_type data will be added for all of these fields.
+            """,
+            prompt_title="Do you wish to process DomainInformation?",
+        )
+        logger.info("Updating DomainInformation(s)...")
+
+        self.update_domain_informations(domain_infos, debug)
+
+    def read_election_board_file(self, domain_election_board_filename):
+        """
+        Reads the election board file and adds each parsed domain to self.domains_with_election_boards_set.
+        As previously implied, this file contains information about Domains which have election boards.
+
+        The file must adhere to this format:
+        ```
+        domain1.gov
+        domain2.gov
+        domain3.gov
+        ```
+        (and so on)
+        """
+        with open(domain_election_board_filename, "r") as file:
+            for line in file:
+                # Remove any leading/trailing whitespace
+                domain = line.strip()
+                if domain not in self.domains_with_election_boards_set:
+                    self.domains_with_election_boards_set.add(domain)
+
+    def update_domain_requests(self, domain_requests, debug):
+        """
+        Updates the organization_type for a list of DomainRequest objects using the `sync_organization_type` function.
+        Results are then logged.
+        Debug mode provides detailed logging.
+        """
+        for request in domain_requests:
+            try:
+                if request.generic_org_type is not None:
+                    domain_name = request.requested_domain.name
+                    request.is_election_board = domain_name in self.domains_with_election_boards_set
+                    request = self.sync_organization_type(DomainRequest, request)
+                    self.request_to_update.append(request)
+
+                    if debug:
+                        logger.info(f"Updating {request} => {request.organization_type}")
+                else:
+                    self.request_skipped.append(request)
+                    if debug:
+                        logger.warning(f"Skipped updating {request}. No generic_org_type was found.")
+            except Exception as err:
+                self.request_failed_to_update.append(request)
+                logger.error(err)
+                logger.error(f"{TerminalColors.FAIL}" f"Failed to update {request}" f"{TerminalColors.ENDC}")
+
+        # Do a bulk update on the organization_type field
+        ScriptDataHelper.bulk_update_fields(
+            DomainRequest, self.request_to_update, ["organization_type", "is_election_board", "generic_org_type"]
+        )
+
+        # Log what happened
+        log_header = "============= FINISHED UPDATE FOR DOMAINREQUEST ==============="
+        TerminalHelper.log_script_run_summary(
```

Comment:
**comment** I feel like it's better to be clear here on the skipped ones why they are being skipped and would want that printed as a reminder so I don't have to remember how code works. Curious on @erinysong and @therealslimhsiehdy as they may be backup for running this.